src/main/webapp/scripts/classes/recommended-video-display-class.js

```diff
@@ -0,0 +1,49 @@
+class RecommendedVideoDisplay{
```

Comment:
Super Extreme Nit (no need to change at all): This class seems very re-usable for any other use case besides recommendations. It would be fine it was called VideoDisplay.

Reply:
I named it thinking we might need a page that just displays one video but that's not happening anymore haha. I'll call it VideoListDisplay just so that it's clear there are many videos here